application/controllers/ScheduleController.php

```diff
@@ -92,6 +106,7 @@ public function addAction(): void
     {
         $this->setTitle($this->translate('New Schedule'));
         $form = (new ScheduleForm(Database::get()))
+            ->setShowTimezoneSuggestionInput()
             ->setAction($this->getRequest()->getUrl()->getAbsoluteUrl())
```

Comment:
It's now required to add `->setParam('showCompact')` here, otherwise tabs are visible inside the modal in case of an invalid tz.